app/server/src/controllers/sessionsController.ts

```diff
@@ -3,6 +3,21 @@ import { AppLocals, SessionMetadata } from "../types";
 import { SessionStore } from "../db/sessionStore";
 import { jsonResponseSuccess } from "../jsonResponse";
 
+export const serialiseSession = (session: string | null) => {
+    // On failure to get a session we could return 404 with some
+    // rich error response, but we don't really have nice handlers
+    // set up for this yet. So instead let's return a json null
+    // which is basically the same as what redis is giving us
+    // anyway, and we can look to handle this in the front end.
+    //
+    // The other trick we have is that because we store the session in
+    // the db as a jsonified string, we can't use jsonResponseSuccess
+    // and re-stringify the json, so instead we manually construct a
+    // json string, making sure that the case where we fail to find a
+    // session is reasonable.
+    return `{"status":"success","errors":null,"data":${session || "null"}}`;
```

Comment:
Ha, yeah, I see. You could parse the session but then jsonResponseSuccess would just stringify it for you again when it wrote out the response which would be rather inefficient! Ick. We should make a `jsonStringResponseSuccess` util if we need to do this anywhere else! 

Could use `jsonResponseError` to return a 404 response?

Reply:
I'll tidy this up first thing tomorrow and get it back to you